### DISCUSSION TITLE
Rejet de la création du projet qfieldcloud : API error

### DISCUSSION THREAD

**Discussion by apprenti1973** [opengisch/QField#4082] [Q&A]
I have a project and I want to uplolad my project in my New pro qfieldcloud pricing.
But the problem is that
![20230328_015132.jpg](https://user-images.githubusercontent.com/115406255/228106137-81946d4f-0955-465a-8a02-4a03dc3e70f1.jpg)

**Comment by ZsanettMed** [comment]
Hey @apprenti1973 

Apparently this is a misconfiguration issue. Please check your settings again, surely you have to add a project name (which is missing regarding your screen).

**Reply by apprenti1973** [reply]
Can you explain more please ?
